common/scripts/ensure-consistent-versions.js

```diff
@@ -46,16 +45,6 @@ function ensurePackageVersionsAreIdentical(packages, versions) {
     }
 }
 
-function ensureTelemetryVersionMatches(version) {
```

Comment:
Dropped this check so that `ensure-consistent-versions` can continue to be run as part of CI.

On release branches, the telemetry version is anyway always updated by another script right before we check.